pkg/interop/crypto/crypto.go

```diff
@@ -30,3 +30,19 @@ func Hash256(b []byte) []byte {
 func VerifySignature(msg []byte, sig []byte, pub []byte) bool {
 	return false
 }
+
+// Secp256k1Recover recovers public key from the given signature (r, s) on the
+// given message hash using Secp256k1 elliptic curve. Flag isEven denotes Y's
+// least significant bit in decompression algorithm. This function uses
+// Neo.Cryptography.Secp256k1Recover syscall.
+func Secp256k1Recover(r []byte, s []byte, messageHash []byte, isEven bool) bool {
```

Comment:
It doesn't return `bool` for sure, it's more like `[]byte` and this return type should also be described correctly (what's in there).

Reply:
Sure, just didn't pay attention for this.